src/storage/storageController.ts

```diff
@@ -32,6 +32,11 @@ export type ConnectionsFromStorage = {
   [connectionId: string]: StoreConnectionInfo;
 };
 
+export enum SecretStorageLocation {
+  Keytar = 'vscode.Keytar',
+  SecretStorage = 'vscode.SecretStorage',
+}
```

Comment:
Side note, the main advantage of TS `enum`s over string unions is when used from vanilla JS code, we basically don’t do this anymore so I’d generally recommend using string unions in new code (definitely not a big deal or a blocker)